AnnoyancesFilter/MobileApp/sections/mobile-app_specific.txt

```diff
@@ -8,6 +8,13 @@
 ! HINTS are at the end of Specific section
 !
 ! SECTION: Mobile apps - Specific
+
+m.bilibili.com##.openapp-dialog
+m.bilibili.com##.m-float-openapp.launch-app-btn
+m.bilibili.com##.related-openapp
+m.bilibili.com##.m-video2-awaken-btn
+m.bilibili.com##.nav-open-app-img
+||s1.hdslb.com/bfs/static/jinkela/mstation-video-h5/assets/navOpenApp.png
```

Comment:
`||s1.hdslb.com/bfs/static/jinkela/mstation-video-h5/assets/navOpenApp.png` is not necessary due to `m.bilibili.com##.nav-open-app-img`